tests/suites/test_suite_rsa.function

```diff
@@ -21,6 +21,9 @@ void rsa_invalid_param()
 
     mbedtls_rsa_init(&ctx);
 
+    TEST_EQUAL(mbedtls_rsa_get_padding_mode(&ctx), MBEDTLS_RSA_PKCS_V15);
+    TEST_EQUAL(mbedtls_rsa_get_hash_id(&ctx), MBEDTLS_MD_NONE);
```

Comment:
This is a good test but it's in the wrong function. This is testing the default values in a newly initialized context. It doesn't belong in a function that checks reactions to invalid parameters.